src/js/index.js

```diff
@@ -131,16 +136,12 @@ export const setUI = async function(state, user){
             expandMenu(state);
         }
     });
-};
-
-
-
-
-
-
-
 
+};
 
+const cleanUp = function(){
+    document.querySelector(".contact-list").innerHTML = '';
+};

```

Comment:
Since cleanup has been removed from `auth.js` [here](https://github.com/Yuxuan-Lin/VioLegacy/pull/33/files#diff-b8619401a86f918b35eeddac4068f7f4dffc186feee51ac21346af6211bea488R15), can this function be removed?